src/encoded/tests/test_fixtures.py

```diff
@@ -95,29 +100,38 @@ def test_fixtures2(minitestdata2, testapp):
 
 
 def test_order_complete(app, conn):
```

Comment:
this could maybe use a comment explaining what it's trying to test

Reply:
Probably, but I'm not the one who knows, so since I'm trying to get this closed, I'll pass. I'm adding a comment asking for one later.